shardingsphere-proxy/shardingsphere-proxy-frontend/shardingsphere-proxy-frontend-mysql/src/main/java/org/apache/shardingsphere/proxy/frontend/mysql/err/MySQLErrPacketFactory.java

```diff
@@ -125,4 +126,11 @@ public static MySQLErrPacket newInstance(final Exception cause) {
         }
         return new MySQLErrPacket(1, CommonErrorCode.UNKNOWN_EXCEPTION, cause.getMessage());
     }
+    
+    private static String getErrorMessage(final SQLException cause) {
+        if (null != cause.getNextException() && StringUtils.isBlank(cause.getMessage())) {
+            return cause.getNextException().getMessage();
```

Comment:
What if the next is not the tail of the link?

Reply:
Could we consider aggregating all messages in the exception link?

Reply:
This does not need to be the tail of the link, just get a meaningful exception for user. 
According to all the code of `setNextException`, it makes sense to start from the first next.

Reply:
The number of connections used by the transaction is unknown, if this messages are aggregated, the result may be difficult to read.